driver/src/main/java/com/code_intelligence/jazzer/junit/RegressionTestArgumentProvider.java

```diff
@@ -43,6 +45,15 @@
 class RegressionTestArgumentProvider implements ArgumentsProvider, AnnotationConsumer<FuzzTest> {
   private static final String INCORRECT_PARAMETERS_MESSAGE =
       "Methods annotated with @FuzzTest must take at least one parameter";
+  private static final AtomicBoolean agentInstalled = new AtomicBoolean(false);
+
+  private static void configureAndInstallAgent(ExtensionContext extensionContext) {
+    if (agentInstalled.compareAndSet(false, true)) {
```

Comment:
Is this check necessary, since both methods are idempotent / check their installation state themselves?

Reply:
It's not necessary now, but I'm not sure I won't break idempotency at some point without noticing - there is just too much global state in these functions. Not rerunning them for each test also seems less wasteful than the explicit check.